component/backend/Model/Lifecycle.php

```diff
@@ -39,7 +39,9 @@ public function onAfterBuildQuery(\JDatabaseQuery $query, $overrideLimits = fals
 			$lifecycleUserIDs = $this->getLifecycleUserIDs($when);
 			$db               = $this->container->db;
 
-			$query->where($db->qn('id') . ' IN (' . implode(',', $lifecycleUserIDs) . ')');
+			if ($lifecycleUserIDs) {
```

Comment:
This seems to be an unrelated commit. Please create a new branch in your repository with up to the previous commit. Then send a new PR from that branch.

Tip: please use a different branch for each PR you submit, not your master. Otherwise every commit you make into your master branch ends up being part of your PR.

Reply:
Thanks! Didn’t know. Was already wondering why it added the second one to the same PR. This is not what I wanted :)
Will try to correct it.